### PR TITLE
Changes in gulpfile needed by Rollup bundler

### PR DESCRIPTION
**Pull request by ma2ciek**
Changes in gulpfile needed by https://github.com/ckeditor/ckeditor5-dev-bundler-rollup/pull/38 

Fixes https://github.com/ckeditor/ckeditor5/issues/374.

### CODE REVIEW

File: build-config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/* jshint browser: false, node: true, strict: true */
+'use strict';
+
+/**
+ * @see https://github.com/ckeditor/ckeditor5-dev-bundler-rollup/issues/37 for config options
```
Comment (Reinmar) [inline build-config.js]: If we want to point someone to options documentation then it should be in the README of that package.

File: build-config.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/* jshint browser: false, node: true, strict: true */
```
Comment (Reinmar) [inline build-config.js]: Missing new line after comment.

File: gulpfile.js [hunk 3]
```diff
@@ -126,34 +126,21 @@ gulp.task( 'compile:themes:esnext', ( callback ) => {
...
-
-gulp.task( 'bundle:clean', ckeditor5DevBundler.cleanFromConfig );
-gulp.task( 'bundle:minify:js', ckeditor5DevBundler.minify.jsFromConfig );
-gulp.task( 'bundle:minify:css', ckeditor5DevBundler.minify.cssFromConfig );
-
-// Generates the bundle without minifying it.
-gulp.task( 'bundle:generate',
-	[
-		'bundle:clean',
-		'compile:js:esnext',
```
Comment (Reinmar) [inline gulpfile.js]: Please check whether the `compile:*` tasks couldn't be removed now.

Comment (ma2ciek) [inline gulpfile.js]: `docs-buider` is still using compiler here, but this part of code can be also moved from the main gulpfile, so here we can expose only `docs`.

**Pull request merged**